sdks/python/apache_beam/ml/inference/huggingface_inference.py

```diff
@@ -659,7 +666,7 @@ def _deduplicate_device_value(self, device: str):
               "but GPUs are not available. Switching to CPU.")
           self._load_pipeline_args['device'] = 'cpu'
     else:
-      if device:
+      if current_device:
         _LOGGER.warning(
```

Comment:
After updates, I'm probably actually inclined to just throw here. There's no way this is an intentional valid configuration since we're defaulting `device` to None.

Reply:
makes sense